client/dfclient/include/fov.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
```

Comment:
please indent with tabs

Reply:
Wait, I actually forgot to fix the indentation in the headers

Reply:
Now it's done.